### PR TITLE
Add click handler

### PR DESCRIPTION
**Pull request by matheus-fls**

### CODE REVIEW

File: src/components/App.js [hunk 1]
```diff
@@ -1,20 +1,84 @@
...
+          next: null,
+          operation: null,
+        });
+      } else {
+        this.setState({
+          total: total || next,
+          operation: null,
+        });
+      }
+    }
```
Comment (shloch) [inline src/components/App.js]: most of this logic should be inside the `"src/logic/calculate.js"` . Thats why we have a logic folder. When you'll should call it here

send it there

**Review by shloch — CHANGES_REQUESTED** [review]
## hello @matheus-fls 
Works perfectly, i like your display... its very cool

You have not managed division by ZERO
Screenshot from 2020-01-17 18-50-40

Please solve the issue above, and the one mentioned below, and file for last review hopefully 🙂🙂

**Comment by matheus-fls** [comment]
Hello Shey, thank you very much for your review.

I believe I did all the required changes. If I forgot something or if there are more issues to be solved please let me know.

**Review by bolah2009 — APPROVED** [review]
It's a 👍🏾 from me!

### :heavy_check_mark: Approved!

Good luck with your next project. :star2: 

Cheers
[Bola](https://bolabuari.com/)

**Pull request merged**